content/docs/other-guides/usage-reporting.md

```diff
@@ -20,24 +20,34 @@ command only restarts spartakus with reportUsage set to `false`
 kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
 ```
 
-**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data
-allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
-
-You can improve the quality of the data by giving each Kubeflow deployment a unique id by editing `spartakus.yaml` in your `KF_APP` directory:
-
-```
-apiVersion: v1
-data:
-  usageId: "your usage id"
-kind: ConfigMap
-metadata:
-  labels:
-    kustomize.component: spartakus
-  name: spartakus-parameters
-  namespace: kubeflow
-```
-
-Then deploy your changes:
-```
-kubectl apply -f spartakus.yaml`
+**To prevent Spartakus from being deployed; users should do the following before running `kfctl apply`** 
+- Edit ${KFAPP}/app.yaml
+- Delete the entry in KfDef.Spec.Applications for spartakus
```

Comment:
Add code formatting to the YAML parts of this line.

Reply:
In fact, I wish to left the YAML to code formatting. Is there some wrong here, sorry inconvenience!

Reply:
This looks good, thanks!